internal/rollup/terser-wrapped.js

```diff
@@ -59,20 +59,20 @@ if (inputs.length != 1) {
 const input = inputs[0];
 
 function runterser(inputFile, outputFile, sourceMapFile) {
-  if (DEBUG) console.error(`Minifying ${inputFile} -> ${outputFile} (sourceMap ${sourceMapFile})`);
+  debug(`minifying ${inputFile} -> ${outputFile} (sourceMap ${sourceMapFile})`);
 
   const terserConfig = {
     'sourceMap': {'filename': sourceMapFile},
     'compress': {
       'pure_getters': true,
       'passes': 3,
       'global_defs': {'ngDevMode': false, 'ngI18nClosureMode': false},
-      'keep_fnames': !debug,
-      'reduce_funcs': !debug,
-      'reduce_vars': !debug,
-      'sequences': !debug,
+      'keep_fnames': !DEBUG,
```

Comment:
seems wrong - DEBUG was set from 'DEBUG_LOGS' - shouldn't this one be set from 'DEBUG'?

Reply:
good catch thanks